src/js/components/FormField/FormField.js

```diff
@@ -45,8 +45,23 @@ const FormFieldContentBox = styled(Box)`
   ${props => props.focus && focusStyle({ justBorder: true })}
 `;
 
-const Message = ({ message, ...rest }) => {
+const Message = ({ error, message, ...rest }) => {
+  const theme = useContext(ThemeContext) || defaultProps.theme;
+  const icon = theme.formField.error && theme.formField.error.icon;
+  const iconContainer =
+    theme.formField.error && theme.formField.error.iconContainer;
   if (message) {
+    if (error) {
+      if (icon && typeof message === 'string')
+        return (
+          <Box direction="row" align="start" gap="xsmall">
+            <Box flex={false} {...iconContainer}>
```

Comment:
I don't think we should be including this Box if there isn't and `icon`.

Reply:
Good point, will make an adjustment